examples/02-custom-vpc-cidr-no-nodes.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   name: cluster-2
   region: eu-north-1
 
-vpc: {cidr: 10.10.0.0/16}
+vpc:
+  cidr: 10.10.0.0/16
+  autoAllocatedIPv6: true
```

Comment:
Not sure about the name yet, maybe just `autoIPv6`, but that reads as if it'd enable full IPv6 thing...

Reply:
Maybe be `autoCIDRv6`?

Reply:
I think `autoAllocatedIPv6` is clear and not too long

Reply:
or `autoAllocateIPv6` ?